packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -91,22 +92,30 @@ function Field({
   const { t } = useLocale();
   const [animationRef] = useAutoAnimate<HTMLUListElement>();
 
-  const [options, setOptions] = useState<SelectOption[]>([
-    { placeholder: "< 10", value: "", id: uuidv4() },
-    { placeholder: "10-100", value: "", id: uuidv4() },
-    { placeholder: "100-500", value: "", id: uuidv4() },
-    { placeholder: "> 500", value: "", id: uuidv4() },
-  ]);
+  const options =
+    useWatch({
+      control: hookForm.control,
+      name: `${hookFieldNamespace}.options`,
```

Comment:
Now that we are saving `options` directly. We need to get rid of `selectText` which was used to build options at run time.

`selectText` is legacy from the time when we provided just a textarea to freely add, reorder options as text.


But `selectText` would still be there for backward compatibility i.e. all existing fields would have selectText(and not options) and that should be mapped to options and we would give preference to `options` over `selectText`